examples/tutorials/core_api_pytorch_mnist/model_def_adaptive.py

```diff
@@ -0,0 +1,246 @@
+# In this stage, we perform a hyperparameter search to search for optimal model parameters. You should expect to see a graph in the WebUI 
```

Comment:
Some of these lines are really long.

Please wrap all lines to 100 characters.

(normally our linters would catch this, not sure what's going on there...)